loadtest/main.go

```diff
@@ -178,11 +179,7 @@ func run(config Config) {
 			txBuilder := TestConfig.TxConfig.NewTxBuilder()
 			_ = txBuilder.SetMsgs(msg)
 			seqDelta := uint64(i / 2)
-			SignTx(&txBuilder, key, seqDelta)
 			mode := typestx.BroadcastMode_BROADCAST_MODE_SYNC
-			if j == len(activeAccounts)-1 {
```

Comment:
We don't want to block on the last account, since that will limit throughput. Rather, we should send everything as SYNC (see differences here https://github.com/cosmos/cosmos-sdk/issues/4186#issuecomment-486664720). Otherwise, throughput is limited at the transaction with broadcast-mode=block